introduction/deprecated_features/README.md

```diff
@@ -0,0 +1,30 @@
+# Deprecated Features
+
+This page lists deprecated features in CSM. They are no longer being actively developed and are planned to be removed in a future CSM release.
+This groups the deprecated features by the CSM release in which they were deprecated, in reverse chronological order (the most recently deprecated
+features are listed first).
+
+## Deprecated in CSM 1.3
+
+* [Boot Orchestration Service (BOS)](../../glossary.md#boot-orchestration-service-bos) v1
+  * It is likely that even prior to BOS v1 being removed from CSM, the [Cray CLI](../../glossary.md#cray-cli-cray) will change its behavior when no
```

Comment:
Please add:
  * The release when BOS v1 will be removed has not been specified yet. However, customers should consider  a migration path from BOS v1 to v2.

Reply:
It occurs to me that this statement is true for all deprecated features that haven't yet had their final CSM versions announced. So I added a paragraph at the top of the page which covers this. Please let me know if it covers your concern.